java/flight/flight-core/src/main/java/org/apache/arrow/flight/FlightRuntimeException.java

```diff
@@ -29,7 +29,7 @@
   /**
    * Create a new exception from the given status.
    */
-  FlightRuntimeException(CallStatus status) {
+  public FlightRuntimeException(CallStatus status) {
```

Comment:
This shouldn't need to be public, a CallStatus has a toRuntimeException method already

Reply:
Addressed in commit:
https://github.com/apache/arrow/pull/7994/commits/f60fc4a27cfadedc4583548debde52369c725c4a